packages/core/src/portable-plugin-config.ts

```diff
@@ -145,8 +145,9 @@ export class PortablePluginConfig {
             return buildBroccoli(placeholder);
           case 'htmlbars-parallel':
             return buildHTMLBars(placeholder);
+          default:
+            assertNever(placeholder);
```

Comment:
Re-introducing this safety check. It's intended to error if a new `placeholder.type` ever gets added without adding it to this switch.

Reply:
Yeah, I understood that, but was not able to silence TS complaining about unreachable code. I wonder why TS treats this call in the `default` case differently than below the switch statement, as semantically there are the same. 🤔